students/tbrackney/Lesson7/html_render.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+"""
+File Name: html_render.py
+Author: Travis Brackney
+Class: Python 201 - Self paced online
+Date Created 5/12/2018
+Python Version: 3.6.4
+"""
+
+''' '''
+
+
+class Element():
+    """Class for rendering html page"""
+    tag = None
+    i_string = '    '
+    indent = 0
+    new_lines = True
+    keywords = ''
+
+    def __init__(self, *args, **kwargs):
+        self.content = []
+        if len(args) is not 0:
```

Comment:
This could be `if args:`, but it seems like you should be using a single `content` parameter rather than variable `*args`